chromadb/test/ef/test_spacy_ef.py

```diff
@@ -0,0 +1,26 @@
+import pytest
```

Comment:
Add `spacy = pytest.importorskip("spacy", reason="spacy not installed")` to skip the test if `spacy` is not installed.

Reply:
Resolved